### PR TITLE
Node.js Span Attributes Hotfix

### PR DESCRIPTION
**Pull request by lukebarrett24**

### CODE REVIEW

File: apm-configuration/node-js.mdx
Comment (eduardo-mw) [inline apm-configuration/node-js.mdx:101]: I would just put endpoints. Because you don't instrument spans. Spans are a result of instrumentation

**Review by eduardo-mw — CHANGES_REQUESTED** [review]

**Review by eduardo-mw — APPROVED** [review]

File: apm-configuration/node-js.mdx
Comment (eduardo-mw) [inline apm-configuration/node-js.mdx:102]: Might want to add something isolating/highlighting the `tracker.addAttribute()`

**Review by eduardo-mw — CHANGES_REQUESTED** [review]

**Review by Zamichel — APPROVED** [review]

**Comment by Zamichel** [comment]
@eduardo-mw PR waiting on your approval

**Review by eduardo-mw — APPROVED** [review]

**Pull request merged**
